d2-schemas/src/main/pegasus/com/linkedin/d2/D2TransportClientProperties.pdl

```diff
@@ -141,4 +141,9 @@ record D2TransportClientProperties {
      */
     HTTP_2
   }
+
+  /**
+   * Maximum ratio of retry requests to total requests per client
```

Comment:
Is this config applicable only when retry is enabled? If so, please document that here.
Also why are other retry configs (retry, retryLimit, Interval) etc not part of the client properties?

Reply:
Updated the documentation:
  /**
   * Service desired maximum ratio of retry requests to total requests per client. The retry ratio is applied
   * only when D2 RetryClient is enabled.
   */

Because the other retry configs (retry, retryLimit, updateInterval) are defined on the client-side, in application.cfg. But maxClientRequestRetryRatio is a service-desired property, which is defined on the server-side and pass to the client via zookeeper.